src/main/java/org/example/home_work5/repository/AccountRepo.java

```diff
@@ -0,0 +1,166 @@
+package org.example.home_work5.repository;
+
+import org.example.home_work5.entity.AccountDTO;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountRepo {
+    private static final String FILE_PATH = "src/main/resources/data.csv";
```

Comment:
what if you read file from another device? good practice is to provide needed file path with environment variable